app/modules/individuals/models.py

```diff
@@ -173,29 +200,28 @@ def run_integrity(cls):
     def remove_all_empty(cls):
         # Individual without encounters are an error that should never really happen
         for indiv in Individual.query.filter(~Individual.encounters.any()).all():
-            indiv.delete_from_edm()
             indiv.delete()
 
     def get_sex(self):
```

Comment:
I'm going to take this as an example, can we remove these `obj.get_X()` functions since they just return the attribute as `obj.X`?

Reply:
Fair point but I'd say that is an additional change to the first drop of EDM retirement (I'm trying to avoid polishing this too much before we hit it with QA) as such. Created https://wildme.atlassian.net/browse/DEX-1357 for this

Reply:
I'm ok with a more drop-in replacement.  We can always simplify and reformat our model attributes vs. getters and setters down the road.